packages/components/addon/components/hds/tabs/index.hbs

```diff
@@ -12,8 +17,8 @@
             "hds/tabs/tab"
             didInsertNode=this.didInsertTab
             willDestroyNode=this.willDestroyTab
+            didUpdateNode=this.didUpdateTab
```

Comment:
maybe we can set `didUpdateNode` before the `willDestroyNode`

Reply:
my bad, fixed